app/design/frontend/Magento/luma/etc/view.xml

```diff
@@ -12,9 +12,9 @@
                 <width>140</width>
                 <height>140</height>
             </image>
-            <image id="cart_cross_sell_products" type="thumbnail">
-                <width>200</width>
-                <height>248</height>
+            <image id="cart_cross_sell_products" type="small_image">
```

Comment:
These changes make sense. As Luma theme is developed mostly for demo but not for extending I would ask you to make similar changes for [Blank theme](https://github.com/magento/magento2/blob/2.2-develop/app/design/frontend/Magento/blank/etc/view.xml#L15) that developed to be a parent for custom themes.